### PR TITLE
Add docs subrepo

### PR DESCRIPTION
**Pull request by piyalbasu**
* Adding docusaurus scaffolding and docs content while stripping out unnecessary boilerplate
* moving playground from `extension` to `docs`

### CODE REVIEW

File: @stellar/lyra-api/webpack.config.js [hunk 1]
```diff
@@ -33,6 +34,11 @@ const config = {
   resolveLoader: {
     modules: [path.resolve(__dirname, "../../node_modules")],
   },
+  plugins: [
+    new webpack.DefinePlugin({
+      DEVELOPMENT: false,
+    }),
```
Comment (piyalbasu) [inline @stellar/lyra-api/webpack.config.js:40]: noticed this bug while I was testing importing lyra-api into docs

File: docs/babel.config.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
```
Comment (piyalbasu) [inline docs/babel.config.js:1]: docusaurus boilerplate

File: docs/sidebars.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+const PLAYGROUND_BASE_PATH = "playground";
```
Comment (piyalbasu) [inline docs/sidebars.js:1]: this file can't be TS bc it isn't compiled, it's run straight from node :(

Comment (piyalbasu) [reply docs/sidebars.js:1]: maybe there's a way around this - prob not worth digging into for this PR

File: docs/src/css/custom.css [hunk 4]
```diff
@@ -0,0 +1,25 @@
+/* stylelint-disable docusaurus/copyright-header */
```
Comment (piyalbasu) [inline docs/src/css/custom.css:1]: docusaurus boilerplate - this comes part of the "classic" theme

File: docs/src/pages/index.tsx [hunk 5]
```diff
@@ -0,0 +1,10 @@
+// @ts-nocheck
+import React from "react";
+
+// Webpoack alias issue with @docusarus
+// eslint-disable-next-line import/no-unresolved
+import { Redirect } from "@docusaurus/router";
```
Comment (piyalbasu) [inline docs/src/pages/index.tsx:6]: there's a known issue where some repo setups load this module fine, but ES & TS linters aren't able to find it. Will hopefully be fixed by Docusaurus soon

File: docs/src/pages/styles.module.css [hunk 6]
```diff
@@ -0,0 +1,37 @@
+/* stylelint-disable docusaurus/copyright-header */
```
Comment (piyalbasu) [inline docs/src/pages/styles.module.css:1]: more docusaurus boilerplate theme styles

File: docs/docs/playground/getPublicKey.mdx [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+#### `getPublicKey()`
+
+<strong>Test Lyra's `getPublicKey` method:</strong>
+
+import { useState } from "react";
+import { getPublicKey } from "@stellar/lyra-api";
+import { PlaygroundInput } from "./basics";
+
+export const GetPublicKeyTest = () => {
```
Comment (piyalbasu) [inline docs/docs/playground/getPublicKey.mdx:14]: some "magic" happens in mdx. For some reason, you have to export this component even though we're instantiating it right below 🤷

File: docs/docusaurus.config.js [hunk 8]
```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  title: "Lyra Documentation",
+  url: "https://lyradocs.netlify.com",
```
Comment (piyalbasu) [inline docs/docusaurus.config.js:3]: will setup this job in the next PR

File: docs/src/css/custom.css (hunk 4 above)
Comment (piyalbasu) [reply docs/src/css/custom.css:1]: i didn't really do any design work, i just swapped in Lyra "purple" into the classic theme.

We can do some work centralizing styles between /docs and /extension in the future, but didn't want to get into that now

File: docs/docs/playground/isConnected.mdx [hunk 9]
```diff
@@ -0,0 +1,32 @@
+---
+id: isConnected
+title: isConnected
+---
+
+#### `isConnected()`
+
+<strong>Test Lyra's `isConnected` method:</strong>
+
+import { useState } from "react";
```
Comment (jeesunikim) [inline docs/docs/playground/isConnected.mdx:10]: I think these need ``` codeblock. I am getting `Can't resolve '@stellar/lyra-api' in '/Users/jeesunkim/Code/lyra/docs/docs/playground'`

Comment (piyalbasu) [reply docs/docs/playground/isConnected.mdx:10]: ah good catch - you actually need to build `@stellar/lyra-api` locally as these playground pages run the `@stellar/lyra-api` library. Added this to README:
```
First build `@stellar/lyra-api` (a dependency of this project) by cd'ing to `@stellar/lyra-api` and running `yarn build`.

Then, come back to this directory and run:

```
$ yarn start
```
```

File: docs/README.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+### Local Development
+
+First build `@stellar/lyra-api` (a dependency of this project) by cd'ing to `@stellar/lyra-api` and running `yarn build`.
+
+Then, come back to this directory and run:
+
+```
+$ yarn start
+```
+
```
Comment (piyalbasu) [inline docs/README.md:14]: just created a ticket to figure out how to automatically build needed subrepos on `yarn start`, but for now we can manually build it

Comment (vcarl) [reply docs/README.md:14]: Yeah development experience is getting a little rough. Should spend some time on that soon

File: docs/docs/playground/getPublicKey.mdx (hunk 7 above)
Comment (vcarl) [reply docs/docs/playground/getPublicKey.mdx:14]: Hm defining a component within MDX feels weird? Why not as a .tsx file that gets imported?

File: docs/docs/playground/basics/index.tsx [hunk 11]
```diff
@@ -0,0 +1,18 @@
+import styled, { css } from "styled-components";
```
Comment (vcarl) [inline docs/docs/playground/basics/index.tsx:1]: IMO we shouldn't default to `index.ts[x]` for first files, I personally find it harder to navigate a codebase with a large number of `index` filenames

File: docs/docs/playground/getPublicKey.mdx (hunk 7 above)
Comment (piyalbasu) [reply docs/docs/playground/getPublicKey.mdx:14]: yeah good thinkin

Comment (piyalbasu) [reply docs/docs/playground/getPublicKey.mdx]: just added a separate folder for the component parts of these mdx files

**Pull request merged**